src/openrct2/ride/TrackPaint.cpp

```diff
@@ -987,12 +987,12 @@ constexpr CoordsXY defaultRightHelixUpSmallQuarterBoundLengths[4][3][2] = {
 
 constexpr CoordsXYZ defaultRightHelixUpSmallQuarterBoundOffsets[4][3][2] = {
     {
-        { { 0, 6, 0 }, { 0, 0, 0 } },
+        { { 0, 6, 7 }, { 0, 0, 0 } },
```

Comment:
Why are these 7 and the rest 6?

Reply:
I tried 6 but then the sprites would still show under the tunnel rather than inside it. It's the lowest I can go. Same with the rest, where 6 was the lowest I could go. I can up all the others to 7 if you'd like for consistency sake, it wont change anything functionality wise.

Reply:
I would prefer consistency across everything so if that means 7 use 7 for everything.

Reply:
Gotcha! I'll do that first thing in the morning then. I'll mark as resolved once I've done that.